apps/docs/self-hosting/deploy/manual.mdx

```diff
@@ -43,7 +43,7 @@ cp .env.example .env
 
 <Note>
   The database user should have the `SUPERUSER` role. You can setup and migrate
-  the database with the `pnpm prisma generate && pnpm db:migrate` command.
+  the database with the `pnpm db:generate && pnpm db:migrate` command.
```

Comment:
In fact, I overlooked, the generate step can be removed as it will be automatically generate during build step.

```suggestion
  the database with the `pnpm db:migrate` command.
```